### PR TITLE
Remove CHAR and FLOAT from Hive product tests

### PR DESCRIPTION
**Pull request by maciejgrzybek**
Appropriate tests will go in along with corresponding patches introducing those data types.

This allows us to unquarantine data type tests for Hive.

TD's PR: https://github.com/Teradata/presto/pull/278

### CODE REVIEW

**Comment by maciejgrzybek** [comment]
Re-pushed (fixed typo in commit msg).

**Comment by maciejgrzybek** [comment]
Small fix to enable some quarantined product tests
Can you, @electrum review or dispatch further?

**Comment by maciejgrzybek** [comment]
Added 6ee1c7e to cover problems with executing product tests on Hive pre 1.1.0 (e.g Hortonworks 2.2 distribution).

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java [hunk 1]
```diff
@@ -42,9 +42,6 @@ private AllSimpleTypesTableDefinitions()
     @TableDefinitionsRepository.RepositoryTableDefinition
     public static final HiveTableDefinition ALL_HIVE_SIMPLE_TYPES_RCFILE = allHiveSimpleTypesTableDefinition("RCFILE", Optional.of("SERDE 'org.apache.hadoop.hive.serde2.columnar.ColumnarSerDe'"));
 
-    @TableDefinitionsRepository.RepositoryTableDefinition
-    public static final HiveTableDefinition ALL_HIVE_SIMPLE_TYPES_KNOWN_TO_PRESTO_TEXTFILE = allHiveSimpleTypesKnownToPrestoTextfileTableDefinition();
```
Comment (raghavsethi) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: Maybe *supported by Presto*?

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java [hunk 2]
```diff
@@ -56,47 +53,37 @@ public Requirement getRequirements(Configuration configuration)
...
                 "9223372036854775807, " +
-                "123.345, " +
                 "234.567, " +
+                "CAST(346 as DECIMAL(10,0))," +
+                "CAST(345.67800 as DECIMAL(10,5))," +
                 "timestamp '2015-05-10 12:15:35.123', " +
                 "date '2015-05-10', " +
                 "'ala ma kota', " +
-                "'ala ma kota', " +
+                "'ala ma kot', " +
```
Comment (raghavsethi) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java]: Revert this change

File: presto-product-tests/src/main/java/com/facebook/presto/tests/TestGroups.java [hunk 3]
```diff
@@ -47,6 +47,7 @@
     public static final String HDFS_IMPERSONATION = "hdfs_impersonation";
     public static final String HDFS_NO_IMPERSONATION = "hdfs_no_impersonation";
     public static final String AUTHORIZATION = "authorization";
+    public static final String HIVE_1_1_0 = "hive_1_1_0";
```
Comment (raghavsethi) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/TestGroups.java]: I think it might be more appropriate to call it 'pre-hive_1_1'? Otherwise it seems like the tests are *only* for Hive 1.1.0.

**Comment by raghavsethi** [comment]
Looks good with some small changes requested.

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java (hunk 2 above)
Comment (maciejgrzybek) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java]: The test was broken and unmaintained for a while (that's why was quarantined). I didn't think it's worth to break it into a separate commit.

**Comment by maciejgrzybek** [comment]
Fixed! Back to you @raghavsethi.

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java (hunk 2 above)
Comment (raghavsethi) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java]: *Prerequisite
*Product

Perhaps: Fix broken product test in quarantined group

**Comment by raghavsethi** [comment]
Merged, thanks!
